.github/workflows/beaker.yml

```diff
@@ -152,6 +152,12 @@ jobs:
     name: "${{ matrix.name }}"
     steps:
       - uses: actions/checkout@v4
+      - name: Setup podman
+        if: ${{ inputs.beaker_hypervisor == 'docker_podman' }}
```

Comment:
Do we actually support `docker_podman` as a type? How it work is:
https://github.com/voxpupuli/beaker/blob/9e1013356cb06cca682c8e385c7232c7c64502cc/lib/beaker/hypervisor.rb#L30-L32

So that translates to `require "beaker/hypervisor/docker_podman"`, which AFAIK doesn't exist. At least https://github.com/voxpupuli/beaker-docker/blob/master/lib/beaker/hypervisor/docker_podman.rb is a 404.

Reply:
Yeah, that still needs creation. Otherwise there is no way to distinguish docker and podman, as they are in the same docker.rb

Reply:
I guess the primary question should be "do we want people to be able to select docker vs podman" (so creating a `docker_podman.rb` with essentially `module Beaker, class DockerPodman < Beaker::Docker, end, end` in it) or should we default to podman and just ignore "real" docker.

Personally, I think letting people select is better.

Reply:
I also think we should use a docker_podman.rb. That enables people to chosse and we've an option to patch code for podman. There are scenarios where it works in a slightly different way, so having an option to apply patches makes sense to me.